edscrapers/transformers/datajson/transform.py

```diff
@@ -119,8 +138,13 @@ def _transform_scraped_resource(target_dept, resource):
     downloadURL = downloadURL.replace(' ','%20')
     distribution.downloadURL = downloadURL
 
-    distribution.title = resource.get('name')
-    distribution.description = resource.get('name')
+    if resource.get('name'):
+        distribution.title = resource.get('name')
```

Comment:
I have an idea - what if we keep a list here with all the super-common extensions (e.g. "Excel", "Doc" etc.) and if a case insensitive match is made, then discard the name and use the file name instead?

Also, instead of regex we could make a Levenshtein threshold of, let's say, one character. What do you think?

Reply:
Started common resource names list with:

```
resources_common_names = ['excel', 'doc', 'download excel', 'download se excel',
                        'dowload doc', 'download this table as an excel file.',
                        'download this table as a microsoft excel spreadsheet',
                        'excel download', 'download standard error excel']
```

If a resource name matches one of this list, then we extract the name from the filename.

Great idea using a Levenshtein threshold. I thinking in use some kind of NLP to extract some fields from the metadata we have, for example, keywords. We can extract keywords from the description if we have a good description. Maybe we can work on these improvements in the next sprint?